src/custom-element.js

```diff
@@ -1665,7 +1668,7 @@ function createBaseCustomElementClass(win) {
     overflowCallback(overflown, requestedHeight, requestedWidth) {
       this.getOverflowElement();
       if (!this.overflowElement_) {
-        if (overflown) {
+        if (overflown && !this.has3pOverflow) {
```

Comment:
And then here:

```js
const overflow = this.getOverflowElement();
if (!overflow) {
  ...
}
```

Reply:
Still need two variable `this.overflowElement_` and `this.has3pOverflow_`. Since it's the 3p context that handle overflow, I don't want runtime to deal with `this.overflowElement_` in that case. 

If so I feel the only difference between the two approaches is where the `this.has3pOverflow` variable is placed, in customElement class or baseElement class? Since it's only a boolean I don't want to write get/set function for the value. 

Should I store the variable to baseElement class instead?

Reply:
You could infer if `getOverflowElement() && !this.overflowElement_`, but that's confusing. 😄 

Ok this is fine, but how about we call it `warnOnMissingOverflow` instead?

Reply:
: ) Renamed